test/stablehlo/test_exports.py

```diff
@@ -26,7 +36,31 @@ def test_interpolate(self):
     ans = model(*arg)
 
     with torch.no_grad():
-      exported = torch._export.export(model, arg)
+      exported = torch.export.export(model, arg)
+      shlo = exported_program_to_stablehlo(exported)
+      ans2 = shlo(*arg).cpu().to(torch.float32)
+      self.assertTrue(torch.allclose(ans, ans2, atol=1e-5))
```

Comment:
What about we set `torch.manual_seed` in this test file in case the test becomes flaky?